subgraphs/uniswap-v3/src/common/price/price.ts

```diff
@@ -171,7 +166,8 @@ export function findUSDPricePerToken(
             nativeTokenValueLocked.gt(largestNativeTokenValue) &&
             nativeTokenValueLocked.gt(
               NetworkConfigs.getMinimumLiquidityThreshold()
-            )
+            ) &&
+            token0.lastPriceUSD != BIGDECIMAL_ZERO
```

Comment:
This doesn't change anything, right?
If this branch (or the one above) doesn't enter we return the default `priceSoFar` at line 182, which will be 0.

If lastPriceUSD is 0 we'll be returning 0 too by multiplying by 0.

Reply:
Correct! I have removed.